README.md

```diff
@@ -1,101 +1,137 @@
+    D4 --> E1
+    E1 --> E2
+    E2 --> E3
+    E3 --> F1
+    F1 --> F2
+    F2 --> F3
+    C6 --> G1
+    G1 --> G2
 ```
 
-Prior to [`@looksrare/sdk-v2@0.9.2`](https://www.npmjs.com/package/@looksrare/sdk-v2/v/0.9.2?activeTab=readme), LooksRareSDK was making batched calls using `0xsequence/multicall`. But this is not natively supported since `@looksrare/sdk-v2@1.0.0`.
-
-## Documentation
-
-Read the [guides](./guides) if you need help with the implementation.
-
-You can also read the detailed [api documentation](./doc).
-
-## FAQ
-
-### ❓ How to use ABIs
-
-The SDK exposes most of the LooksRare contract [ABIs](https://github.com/LooksRare/sdk-v2/tree/master/src/abis). For convenience, some commonly used ABIs are also exported.
-
-```js
-import LooksRareProtocolABI from "@looksrare/sdk-v2/dist/abis/LooksRareProtocol.json";
-```
-
-### ❓ How to retrieve order nonce ?
-
-Call the public api endpoint [/orders/nonce](https://looksrare.dev/reference/getordernonce), and use this nonce directly.
-
-### ❓ What to do when the order is created and signed ?
-
-Use the public api endpoint [/orders](https://looksrare.dev/reference/createorder) to push the order to the database. After that, the order will be visible by everyone using the API (looksrare.org, aggregators, etc..).
-
-### ❓ When should I use merkle tree orders ?
-
-Merkle tree orders are used to create several orders with a single signature. You shouldn't use them when using a bot. Their main purpose is to facilitate orders creation using a user interface.
-
-### ❓ Why do I need to call grantTransferManagerApproval ?
-
-When you approve a collection to be traded on LooksRare, you approve the TransferManager instead of the exchange. Calling `grantTransferManagerApproval` gives the exchange contract the right to call the transfer function on the TransferManager. You need to call this function only once, the first time you use the V2.
-
-### ❓ What are subset nonces and how to use them ?
-
-tl;dr subset nonces allow you to cancel all the orders sharing the same subset nonce.
-Subset nonces allow you to group some orders together according to arbitrary rules (for example all your orders for a specific collection, all your orders above a certain threshold, etc). You can then cancel them all with a single call to `cancelSubsetOrders`.
-:information_source: Unlike order nonces, executing an order with a specific subset nonce doesn't invalidate other orders sharing the same subset nonce.
-
-## Resources
-
-🔗 [Developer documentation](https://docs.looksrare.org/developers/welcome)
-
-🔗 [Public API documentation](https://looksrare.dev/reference/important-information)
+### Data
 
-🔗 [Developer discord](https://discord.gg/jJA4qM5dXz)
+Order information and nonces live in the `data-staging` and `data-production` database. Nonces are invalidated on-chain, but keeping track of the current nonce for a user happens off-chain in the DB. Orders do not live on-chain, but as they are signed they can be verified.
```

Comment:
Don't think this is relevant for a readme. If somebody forks this and wants to use a differen DB setup that's fine

Reply:
Addressed